index.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Youtube clone</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <!-- I suggest we use the flex displaying to better handle the direction and sizes of boxes -->
+    <header>
+    <nav>
+      <!-- In the navbar, we first have image, the first is the 3 bands and the secon is the logo -->
+      <div class="logo-container">
+        <a href="#subscribe"><img src="./img/humburgerIcon.png" alt="Menu icon" class="menu"/></a>
+        <img src="./img//youtube1.png" alt="youtube logo" height="20px" class="logo" />
+      </div>
+      <input
+        class="search"
+        type="text"
+        name="search"
+        placeholder="Search"
+      /><span><button class="btn-search">Search</button></span>
+      <div class="btn-section">
+        <div style="width: 50px"></div>
+        <img src="./img/videoIcon.png" alt="icons" class="icons" />
+        <img src="./img/menuIcon.png" alt="icons" class="icons" />
+        <img src="./img/notifIcon.png" alt="icons" class="icons" />
+        <img src="./img/icons8-male-user-100.png" alt="avater" class="avatar" />
+      </div>
+    </nav>
+    </header>
+    <main>
+    <div class="pageContent">
+        <div class="content">
+          <div class="VideoContainer">
+            <video width="100%" height="400" controls>
+              <source src="movie.mp4" type="video/mp4" />
+              <source src="movie.ogg" type="video/ogg" />
```

Comment:
These video sources you used here are not working, please make sure that you use real links to real video from youtube, I would sugest you use the `<iframe/>` tag with the embedded video from the real youtube link

Reply:
Thank you for your suggestion, all of them are fixed, please take a look.